packages/artifactor/index.ts

```diff
@@ -47,6 +50,17 @@ class Artifactor {
       const tmpArtifactArray = artifactObjects;
 
       tmpArtifactArray.forEach(artifactObj => {
+        if (newArtifactObjects[artifactObj.contract_name]) {
+          if (this.errorOnDuplicates) {
+            throw new Error(
+              `Duplicate contract names found for ${artifactObj.contract_name}.`
+            );
+          } else {
+            console.warn(
```

Comment:
Let's just make this call happen when a duplicate name is found. How would you feel about the following messaging and formatting?
```
`${OS.EOL}> Duplicate contract names found for ${artifactObj.contract_name}.${OS.EOL}` +
`> This can cause errors and unknown behavior. Please rename one of your contracts.`
```